api.json

```diff
@@ -4179,71 +4610,17 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/BadRequestError"
-                }
-              }
-            }
-          }
-        },
-        "security": [
-          {
-            "app_key": []
-          }
-        ]
-      }
-    },
-    "/notifications/{notification_id}/export_events?app_id={app_id}": {
-      "post": {
-        "operationId": "export_events",
-        "summary": "Export CSV of Events",
-        "description": "Generate a compressed CSV report of all of the events data for a notification.\nThis will return a URL immediately upon success but it may take several minutes for the CSV to become available at that URL depending on the volume of data. Only one export can be in-progress per OneSignal account at any given time.",
-        "parameters": [
-          {
-            "name": "notification_id",
-            "in": "path",
-            "required": true,
-            "description": "The ID of the notification to export events from.",
-            "schema": {
-              "type": "string"
-            }
-          },
-          {
-            "name": "app_id",
-            "in": "query",
-            "required": true,
-            "description": "The ID of the app that the notification belongs to.",
-            "schema": {
-              "type": "string"
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/ExportEventsSuccessResponse"
-                }
-              }
-            }
-          },
-          "400": {
-            "description": "Bad Request",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/BadRequestError"
+                  "$ref": "#/components/schemas/GenericError"
                 }
               }
             }
           },
-          "404": {
```

Comment:
Looks like we are moving the 404 error here, I believe we should keep it as it could still happen on this end point.

Reply:
Done.